synapse/models/person.py

```diff
@@ -63,7 +63,7 @@ def getBaseModels():
                 ('ps:hasalias', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|alias,ps:name'}),
                 ('ps:hasphone', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|phone,tel:phone'}),
                 ('ps:hasemail', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|email,inet:email'}),
-                ('ps:hasnetuser', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|netuser,inet:netuser'}),
+                ('ps:hasnetuser', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|netuser,inet:web:acct'}),
```

Comment:
Same as comment on orgs.py - is it worth changing ps:hasnetuser to ps:haswebacct?

Reply:
Same as above